### DISCUSSION TITLE
collection tag "is_past" and "is_before" not working

### DISCUSSION THREAD

**Discussion by pathly-dev** [statamic/cms#6750] [Troubleshooting]
Hey there, I'm not sure if it's me not knowing how to use it or a bug.
I'm trying to filter my collection entries by date. I tried two different things:

First, I tried to only show entries where the date is in the past, like so:
```
{{ collection:stories limit="6" sort="release_date:desc" release_date:is_past="true" }}
        {{ partial:card-slider }}
{{ /collection:stories }}
```
But if it's true, then every entry (also with date in the future) is shown. If it's false, then nothing is shown.
I also tried with the colon before: `:release_date:is_past="true"`, not sure when to use colon and when not.

Second, I tried `is_before`, like so:
```
{{ collection:stories limit="6" sort="release_date:desc" release_date:is_before="{today}" }}
        {{ partial:card-slider }}
{{ /collection:stories }}
```
Is also tried `release_date:is_before="today"`, `release_date:is_before="{now}"`, `release_date:is_before="now"`, and same with colon: `:release_date:is_before="{today}"` etc. But nothing is working. It always shows everything. 
When I'm trying `is_after` as opposite, then nothing is shown anymore.

**Comment by edalzell** [comment]
are you using a dated collection?

**Reply by pathly-dev** [reply]
What does that mean? 
I have a field `release_date` of type date.

**Comment by pathly-dev** [comment]
Anyone?

**Reply by edalzell** [reply]
You only want to see entries with `release_date` in the past? i.e. before today?

**Reply by pathly-dev** [reply]
@edalzell in the past + today

**Reply by edalzell** [reply]
Try this:

```
    {{ collection:blog :release_date:is_before="now | format('Y-m-d H:i:s')" }}
        <p>{{ title }}</p>
    {{ /collection:blog }}
```

**Reply by jasonvarga** [reply]
Just do `release_date:is_before="now"` and forget the `:` prefix.

**Reply by pathly-dev** [reply]
@jasonvarga I already tried this, but this doesnt work. The entries with date in the future are still shown.
